### DISCUSSION TITLE
DISMTools v.Any General Thread - For things not requiring their own thread.

### DISCUSSION THREAD

**Discussion by AnubyteCode** [CodingWonders/DISMTools#139] [General]
**Just thought one of these could be useful, to minimize clutter.**

I was curious, as I haven't been able to dig into the whole system yet. Is there a method to install .mum files directly? If not, I have a batch file that could be incorporated/edited/cut to help function in such a role. It handles offline and online images, mounting, unmounting, and the existence of an update.mum (to use it or ignore it).
https://gist.github.com/AnubyteCode/4441a18b44be9a19c418cc7271e30ac1

**Comment by CodingWonders** [comment]
@AnubyteCode, DISMTools already supports adding CAB and MSU packages, both of which have MUM files. The latter contains the CAB files necessary for addition.

While this is a cool idea, isn't it a bit redundant?

**Reply by AnubyteCode** [reply]
There is a community of people that edit their own update packages or only add the .mums they want. Thought about them when suggesting the idea. It's admittedly an advanced user feature.

**Reply by CodingWonders** [reply]
@AnubyteCode, I'll see what I can do then

**Reply by AnubyteCode** [reply]
Cool, thanks. Absolutely no rush. Perhaps in an "advanced uses: know what you are doing, risks ahead" section.

Again, thanks for the great work. When I finish my tasks with nvidiaprofileinspector and clean up uupmediacreator I'll see if I can find something to add to dismtools myself. Little things, likely. Still a noob by most standards.

**Reply by CodingWonders** [reply]
Initial work on the MUM package addition procedure has started, and has caused no problems with other package addition methods so far:

![DISMTools_JDKgSCWsgF](https://github.com/CodingWonders/DISMTools/assets/101426328/10b276c2-ce85-4595-9815-7be87e336a57)

![DISMTools_niSXSYNwkI](https://github.com/CodingWonders/DISMTools/assets/101426328/6b4d48f5-f39f-46f3-9fbd-20e40eb36cbb)

**Comment by jekovcar** [comment]
Is it possible to use the settings in a project for another image as a template for further import? Thanks!

**Reply by CodingWonders** [reply]
Project settings make a reference to the mounted image. So, if you copy all the project settings from a project that has an image mounted, then the copy will refer to that mount directory and work with it.

What you can do is unmount the image from the original project, then copy project settings somewhere else, and remount the image in the original project.

**Reply by jekovcar** [reply]
But after unmounting, the "project settings" do not contain any changes about the mounted image, isn't it?

**Reply by CodingWonders** [reply]
Correct. When you unmount the image, the information is removed from project settings.

**Reply by jekovcar** [reply]
By the way, the latest option DISMTools v0.7.2_pre 251123 "Delete volume images" does not work on my Win11x64 25H2 b 26200.7171. DISMTools v0.7.1_stable works fine.

**Reply by CodingWonders** [reply]
Create an issue.

**Reply by jekovcar** [reply]
> if you copy all the project settings from a project that has an image mounted, then the copy will refer to that mount directory and work with it.

But copying all the settings of a project >20 GB as a 'template' is, to put it mildly, not very convenient.
Please make at least some export/import of settings for modified elements to automate other projects. Thanks!

**Comment by jekovcar** [comment]
Hi,
Why "Toll Create Disk Image" require Windows ADK (around 4Gb) , when can be get by [MSDN link](https://msdl.microsoft.com/download/symbols/oscdimg.exe/688CABB065000/oscdimg.exe) and it's necessary components as "etfsboot.com" "efisys.bin" are contained in OS disto ?

**Reply by CodingWonders** [reply]
I will not download files from the symbol server. Besides, you need around that much space for both the Windows ADK and the PE add-on.
